src/extends/creep/movement.js

```diff
@@ -4,9 +4,10 @@ const struckThreshold = 2
 const struckRerouteThreshold = 4
 const travelToDefaults = {
   'reusePath': 1500,
-  'maxRooms': 16,
   'moveToOverride': true,
-  'ignoreStuck': false
+  'ignoreStuck': false,
+  'ignoreHostileCities': true,
```

Comment:
No, I'm sticking with the nomenclature used by the existing API. If you set `ignoreHostileCities` to true it will act just like `ignoreCreeps` and won't take them into account at all. If `ignoreHostileCities` is set to `false` then hostile cities will be taken into account when pathfinding.

Reply:
Good.

Reply:
This feature was added in before I did the `findRoute` stuff. Before that this was the only way to avoid hostile rooms, but now the `findRoute` stuff handles that in a much better way. That's why I disabled these features by default. In fact they should only be used when the `direct` option is passed (the `direct` option bypasses `findRoute` to allow the pathfinder to search whatever rooms it wants, so in that case this could add some limits in).

Reply:
Ah nice.